### PR TITLE
added config for grocy

### PR DESCRIPTION
**Pull request by homerr**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: grocy.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        #auth_basic_user_file /config/nginx/.htpasswd;
+
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /login;
+
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
+        set $upstream_grocy grocy;
+        proxy_pass http://$upstream_grocy:80;
```
Comment (MrTarantula) [inline grocy.subdomain.conf.sample]: The LinuxServer readme specifies 9283 as a recommended port (although grocy runs on 80 internally).

Comment (j0nnymoe) [reply grocy.subdomain.conf.sample]: Because when running this reverse proxy and on a bridge, it'll use port 80.

**Comment by Akruidenberg** [comment]
How to get this working? Linuxserver Grocy has allreally an NGINX server inside. Ik would like to use the letsencrypt container for this.

File: grocy.subdomain.conf.sample (hunk 1 above)
Comment (Akruidenberg) [reply grocy.subdomain.conf.sample]: Fixed with 9283 as port.

**Comment by aptalca** [comment]
@homerr I cannot get the subfolder method working no matter what I try. The subdomain works with no problems (without a base url)

**Comment by aptalca** [comment]
@homerr can you remove the subfolder one from this PR and we'll merge the subdomain only? We can then try and figure out how to make subfolder work

**Comment by CHBMB** [comment]
I'm closing this as subfolder isn't working

**Pull request closed without merging**